packages/eslint-config-eslint/package.json

```diff
@@ -36,6 +66,20 @@
     "eslint-plugin-n": "^17.11.1",
     "eslint-plugin-unicorn": "^52.0.0"
   },
+  "devDependencies": {
+    "@arethetypeswrong/cli": "^0.16.4",
+    "eslint": "^9.14.0",
+    "eslint-config-eslint": "file:",
```

Comment:
As explained in https://nodejs.org/api/packages.html#self-referencing-a-package-using-its-name, this dependency is unnecessary: `eslint-config-eslint` will be automatically resolved to the current package because it matches the `"name"` property in `package.json` and an `"exports"` property is present.